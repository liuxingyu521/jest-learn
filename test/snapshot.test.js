/**
 * snapshot 快照比较
 */

it('snapshot compare, must be the same 1', () => {
  const user = {
    id: 8,
    name: 'blob',
  };

  expect(user).toMatchSnapshot();
});

it('snapshot compare, must be the same 2', () => {
  const user = {
    date: new Date(),
    id: 20,
    name: 'Aliced',
  };

  expect(user).toMatchSnapshot({
    date: expect.any(Date),
    id: expect.any(Number)
  });
});