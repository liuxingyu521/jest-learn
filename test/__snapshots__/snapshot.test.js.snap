// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot compare, must be the same 1 1`] = `
Object {
  "id": 8,
  "name": "blob",
}
`;

exports[`snapshot compare, must be the same 2 1`] = `
Object {
  "date": Any<Date>,
  "id": Any<Number>,
  "name": "Aliced",
}
`;
